hopr/hopr-lib/src/chain.rs

```diff
@@ -324,7 +324,6 @@ where
         expected_block_time: Duration::from_millis(chain_config.chain.block_time),
         tx_polling_interval: Duration::from_millis(chain_config.tx_polling_interval),
         finality: chain_config.confirmations,
-        min_block_range_fetch_size: 3,
```

Comment:
Release config change should require a documentation and example.cfg file change potentially?

Reply:
This was never exposed neither CLI nor config.